cocotb/share/lib/simulator/simulatormodule.h

```diff
@@ -130,4 +130,6 @@ static PyMethodDef SimulatorMethods[] = {
     {NULL, NULL, 0, NULL}        /* Sentinel */
 };
 
+PyMODINIT_FUNC PyInit_simulator(void);
+
```

Comment:
Why did we get away with not having this before? Does this need to be `extern "C"`?

Reply:
Looks like python adds the extern C for us.

Reply:
That was a unintended commit missed after a lot of rebasing. It's been removed. This whole header file is pointless...